gammapy/datasets/make.py

```diff
@@ -64,3 +72,147 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory_name, n_obs, debug=False):
+    """Make a test observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory_name : string
+    	name of the observatory; a list of choices is given in `~gammapy.obs.observatory_locations`
+    n_obs : integer
+    	number of observations for the obs table
+    debug : bool
+    	show UTC times instead of seconds after the reference
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # build a time reference as the start of 2010
+    dateref = Time('2010-01-01 00:00:00', format='iso', scale='utc')
+    # TODO: using format "iso" for `~astropy.Time` for now; eventually change it
```

Comment:
I'm actually -1 on using the "fits" format, I think we should use "iso".
There's plenty of software (like python datetime or pandas) that can parse the iso format, the fits format is much less common.
The fact that we use UTC should be stated once in the docs and then it's OK.